less/dropdowns.less

```diff
@@ -182,7 +183,23 @@
   // Show the menu
   > .dropdown-menu {
     display: block;
-    animation: showDropdown 0.2s cubic-bezier( 0.6, 0.2, 0.1, 1 ) both;
+    animation: showDropdown 0.2s cubic-bezier(0.6, 0.2, 0.1, 1) both;
+  }
+
+  @media (max-width: @screen-sm-min) {
+    > .dropdown-menu.dropdown-menu-overlay {
```

Comment:
I don't think you need to qualify it with `.dropdown-menu`, should be fine as `.dropdown-menu-overlay` assuming this follows the other styles.